Apps/W1/HybridGP/app/src/Migration/Vendors/GPVendorMigrator.codeunit.al

```diff
@@ -312,8 +322,32 @@ codeunit 4022 "GP Vendor Migrator"
         FoundKnownCountry: Boolean;
         CountryCodeISO2: Code[2];
         CountryName: Text[50];
+        IsTemporaryVendor: Boolean;
+        HasOpenPurchaseOrders: Boolean;
+        HasOpenTransactions: Boolean;
+        TemporaryVendorMigratedLogTxt: Text[500];
     begin
         VendorNo := CopyStr(GPVendor.VENDORID, 1, MaxStrLen(Vendor."No."));
+
+        if not ShouldMigrateVendor(VendorNo, IsTemporaryVendor, HasOpenPurchaseOrders, HasOpenTransactions) then
+            exit;
+
+        if IsTemporaryVendor then begin
+            TemporaryVendorMigratedLogTxt := 'Temporary vendor was migrated because it ';
+
+            if HasOpenPurchaseOrders and HasOpenTransactions then
+                TemporaryVendorMigratedLogTxt += 'has open POs and open AP transactions.'
```

Comment:
Do not split into multiple sentances, use placeholders Temporary vendor was migrated because %1
%1 should be replaced by other labels or build with commas.